### PR TITLE
[12.0][MIG] purchase_invoicing_no_zero_line

### PR DESCRIPTION
**Pull request by igallart**

### CODE REVIEW

**Comment by kittiu** [comment]
@igallart thanks for this contribution. Following are some initial comments,

* For ease of search, I think name of PR should be as coding standard `[12.0][MIG] purchase_invoicing_no_zero_line`
* It is good time to add tests.
* In manifest, you can add Development Status
* In Contributors file, please add original contributors -> Digital5, S.L. <jonerikceberio@digital5.es>

Thanks again for the work :+1:

File: purchase_invoicing_no_zero_line/readme/CONFIGURE.rst [hunk 1]
```diff
@@ -0,0 +1,8 @@
+An automated task is included to invoice all pending purchase orders every
```
Comment (kittiu) [inline purchase_invoicing_no_zero_line/readme/CONFIGURE.rst:1]: * The word "automated task" make me think about scheduled job.
* Anything to do with "week"? I though this just remove the line when create invoice from PO.

Can you revise this?

**Comment by kittiu** [comment]
@igallart 
Let me know if any help needed to move this forward.

**Comment by igallart** [comment]
> @igallart
> Let me know if any help needed to move this forward.

Hello @kittiu, you can to do anything, I don't have time, I'm sorry.

**Pull request closed without merging**